python/lsst/ts/atbuilding/csc/mock_controller.py

```diff
@@ -0,0 +1,339 @@
+# This file is part of ts_atbuilding_csc
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["MockVentController"]
+
+import asyncio
+import json
+import logging
+import traceback
+from collections import deque
+
+from lsst.ts import tcpip, utils
+from lsst.ts.xml.enums.ATBuilding import FanDriveState, VentGateState
+
+
+def _cast_string_to_type(new_type: type, value: str):
```

Comment:
Is this the same method that is available in the other module? you could import it from there no?